tests/phpunit/elementor/core/utils/test-collection.php

```diff
@@ -410,4 +410,25 @@ public function test_sort_keys() {
 		$this->assertEqualSets( [ 'a' => 1, 'b' => 1, 'c' => 1 ], $result_asc );
 		$this->assertEqualSets( [ 'c' => 1, 'b' => 1, 'a' => 1 ], $result_desc );
 	}
+
+	public function test_each() {
+		// Arrange
+		$collection = new Collection( [ 'a' => 1, 'b' => 2, 'c' => 3 ] );
+
+		$mock = $this->getMockBuilder( \stdClass::class )
+			->setMethods( [ 'each_callback' ] )
+			->getMock();
+
+		// Expect
+		$mock->expects( $this->exactly( 3 ) )
+			->method( 'each_callback' )
+			->withConsecutive(
+				[ 1, 'a' ],
+				[ 2, 'b' ],
+				[ 3, 'c' ]
+			);
+
+		// Act
+		$collection->each( [ $mock, 'each_callback' ] );
```

Comment:
Please create a change on each array item, and assert that the returned collection is updated with these changes.

Reply:
The collection shouldn't be updated. In order to update you should use map()

Reply:
Done,